### PR TITLE
vp9_lp: enable vp9_lp encoding for gst-msdk

### PR DESCRIPTION
**Pull request by wangzj0601**
Signed-off-by: Wang Zhanjun <zhanjunx.wang@intel.com>

### CODE REVIEW

File: test/gst-msdk/encode/vp9.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  @slash.requires(*have_gst_element("msdkvp9dec"))
+  @slash.parametrize(*gen_vp9_cqp_lp_parameters(spec))
+  def test(self, case, ipmode, qp, quality, slices, refmode, looplvl, loopshp):
+    slash.logger.notice("NOTICE: 'refmode' parameter unused (not supported by plugin)")
+    slash.logger.notice("NOTICE: 'looplvl' parameter unused (not supported by plugin)")
+    slash.logger.notice("NOTICE: 'loopshp' parameter unused (not supported by plugin)")
+    vars(self).update(spec[case].copy())
+    vars(self).update(
+      case      = case,
+      gop       = 30 if ipmode != 0 else 1,
```
Comment (uartie) [inline test/gst-msdk/encode/vp9.py]: What about `lowpower = True`?

File: test/gst-msdk/encode/vp9.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    slash.logger.notice("NOTICE: 'refmode' parameter unused (not supported by plugin)")
+    slash.logger.notice("NOTICE: 'looplvl' parameter unused (not supported by plugin)")
+    slash.logger.notice("NOTICE: 'loopshp' parameter unused (not supported by plugin)")
+    vars(self).update(spec[case].copy())
+    vars(self).update(
+      bitrate   = bitrate,
+      case      = case,
+      fps       = fps,
+      frames    = vars(self).get("brframes", self.frames),
+      gop       = gop,
```
Comment (uartie) [inline test/gst-msdk/encode/vp9.py]: What about `lowpower = True`?

File: test/gst-msdk/encode/vp9.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
```
Comment (uartie) [inline test/gst-msdk/encode/vp9.py]: What about `lowpower = True`?

File: test/gst-msdk/encode/vp9.py (hunk 1 above)
Comment (wangzj0601) [reply test/gst-msdk/encode/vp9.py]: according to gst-msdk, it only support vp9 lp mode and it has no para "low-power" in vp9enc, so it does not need this variable.

**Comment by uartie** [comment]
no activity... closing.

**Pull request closed without merging**